spec/till_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'till'
+
+describe Till do
+
+  subject = described_class
+
+  let(:coffee_shop) { {
+    shopName: "Happy Coffee Shop",
+    prices: { "Latte": 4, "Flat White": 3 },
+    taxRate: 0.0864 } }
+
+  let(:customer_order) { {
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.